### PR TITLE
DefectList Modifications done--save button confirmation box issue fixed.

### PR DESCRIPTION
**Pull request by 212040-AnnMariaJenson**

### CODE REVIEW

File: src/pages/compose/index.tsx [hunk 1]
```diff
@@ -81,13 +81,13 @@ const Compose = () => {
     setSaveModalVisible(false);
   };
   const handleSaveClick = () => {
-    if (name.trim() === "") {
+    if (name && name.trim() === "") {
```
Comment (sourcecoursepoc) [inline src/pages/compose/index.tsx]: Add conditional checking like name?.trim()

File: src/pages/compose/index.tsx [hunk 2]
```diff
@@ -81,13 +81,13 @@ const Compose = () => {
...
       setNameError(true);
     } else {
       setNameError(false);
     }
 
-    if (description.trim() === "") {
+    if (description &&description.trim() === "") {
```
Comment (sourcecoursepoc) [inline src/pages/compose/index.tsx]: Conditional checking here also

**Review by sourcecoursepoc — CHANGES_REQUESTED** [review]

File: src/pages/compose/index.tsx (hunk 2 above)
Comment (212040-AnnMariaJenson) [reply src/pages/compose/index.tsx]: Done

File: src/pages/compose/index.tsx (hunk 1 above)
Comment (212040-AnnMariaJenson) [reply src/pages/compose/index.tsx]: Done

**Review by sourcecoursepoc — APPROVED** [review]

**Pull request merged**